generic/service_check.py

```diff
@@ -36,8 +37,13 @@ def test(self):
         services_list = parser.get(detected_distro.name, 'services').split(',')
         if 'PowerNV' in open('/proc/cpuinfo', 'r').read():
             services_list.extend(['opal_errd', 'opal-prd'])
+            if os.path.exists('/proc/device-tree/bmc'):
+                services_list.remove(['opal_errd'])
         else:
             services_list.extend(['rtas_errd'])
+        if 'Ubuntu' in detected_distro.name:
+            if detected_distro.version >= 17:
```

Comment:
@abdhaleegit 17 or 18 ?

Reply:
netplan is supported as on 17.10

Reply:
@abdhaleegit and what about 17.04 ?

Reply:
@narasimhan-v I mean nteplan supports from 17.04, 17.10 and later